.github/workflows/ci.yml

```diff
@@ -0,0 +1,119 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - "release/**"
+  pull_request:
+
+jobs:
+  lint-rust:
+    name: Rust Style/Linting
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

Comment:
I am a little curious how this can compile clippy without submodules. I'm guessing it may not run the build script, but can you confirm this?